### ISSUE TITLE
[tooltip]: Does not handle accessibility out of the box

### ISSUE THREAD

**Issue by jorenbroekema** [ing-bank/lion#451]
in the config, isTooltip and handlesAccessibility options are not passed, so they are false by default.

Should probably set both to true, so we get the describedby out of the box. And add tests for this.

**Comment by daKmoR** [comment]
@jorenbroekema I can't really follow? what is the issue here - can you maybe rephrase?

**Comment by jorenbroekema** [comment]
In the OverlayController config we have 2 boolean options "handlesAccessibility" and "isTooltip", which if true will configure a few things that improves a11y. Currently I think for lion-tooltip they are not enabled while they should? At least that was true when I started the issue.

**Comment by jorenbroekema** [comment]
https://github.com/ing-bank/lion/issues/269 this is still remaining, but other than that, this issue is solved now
